server/routes/users.js

```diff
@@ -2,14 +2,272 @@
 var express = require('express');
 var router = express.Router();
 
+
+// Require database adapter file (not node-postgres directly)
+const db = require('../db')
+
 // Returns test string to verify that Users server is running.
 router.get('/ping', function(req, res, next) {
   res.send('pong - Users API');
 });
 
 // GET users listing.
 router.get('/', function(req, res, next) {
-    // TODO
+    db.query('SELECT id FROM swe_user', [], (err, result) => {
+    if (err) {
+        return next(err);
+    }
+    res.send(result.rows);
+  });
+});
+
+
+//having an issue with concat
+/*
+//GET all user names.
+router.get('/', function(req, res, next) {
+    db.query('SELECT CONCAT(first_name, ' ', last_name) FROM swe_user', [], (err, result) => {
+    if (err) {
+        return next(err);
+    }
+    res.send(result.rows);
+  });
+});
+*/
+
+
+//GET all user emails.
```

Comment:
Styling comment: there should be a space between '//' and the comment text

I like how you used CAPS with the http method, try to do this consistently in your comments. (e.g. do GET instead of Get)